L1Trigger/L1TTwinMux/interface/DTRPCBxCorrection.h

```diff
@@ -0,0 +1,64 @@
+//-------------------------------------------------
+//
+//   Class: DTRPCBxCorrection
+//
+//   DTRPCBxCorrection
+//
+//
+//   Author :
+//   G. Flouris               U Ioannina    Mar. 2015
+//--------------------------------------------------
+
+#ifndef DTRPCBxCorrection_H
+#define DTRPCBxCorrection_H
+
+#include "DataFormats/L1DTTrackFinder/interface/L1MuDTChambPhContainer.h"
+#include "DataFormats/L1DTTrackFinder/interface/L1MuDTChambThContainer.h"
+
+#include "CondFormats/L1TObjects/interface/L1TwinMuxParams.h"
+#include "CondFormats/DataRecord/interface/L1TwinMuxParamsRcd.h"
+#include "L1Trigger/L1TTwinMux/interface/L1MuTMChambPhContainer.h"
```

Comment:
Is this one different from DataFormats/L1DTTrackFinder/interface/L1MuDTChambPhContainer.h ?
If it is a data format it should go in the DataFormats/ package. There either in L1TMuon or in a new directory L1TTwinMux. @rekovic what is your opinion on this?